windows/src/desktop/kmshell/locale/kan-Knda-IN/locale.xml

```diff
@@ -0,0 +1,307 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE Locale SYSTEM "../../xml/localedef.dtd">
+<Locale Version="10.0.1100.0" OnlineProductID="30">
+  <String Type="PlainText" Id="S_LocaleAuthors">www.ಸಿರಿಗನ್ನಡ.com (www.sirigannada.com)</String>
```

Comment:
Can we make `www.sirigannada.com` the primary link? `www.ಸಿರಿಗನ್ನಡ.com` doesn't resolve for me.

Reply:
It's not actually a link, it's a name... so I don't think we need to worry about it.